validator/tasks/scheduler.py

```diff
@@ -27,13 +27,10 @@ async def _get_models(keypair: Keypair) -> AsyncGenerator[str, None]:
         raise TypeError(
             "Expected a list of responses from GET_ALL_MODELS_ENDPOINT")
     models: list[dict[str, Any]] = response
-    TEMP_MODEL_FAMILIES_ACCEPTED = [
-        "qwen", "llama", "falcon", "mistral", "gemma", "gemini", "phi"]
-    model_ids = [
-        model.get(csts.GET_ALL_MODELS_ID, "")
-        for model in models
-        if any(family in model.get(csts.GET_ALL_MODELS_ID, "").lower() for family in TEMP_MODEL_FAMILIES_ACCEPTED)
-    ]
+
+    model_ids = [model[csts.GET_ALL_MODELS_ID]
```

Comment:
GET_ALL_MODELS_ID should be called MODEL_ID

after the check could be

```python
model_ids = model[cst.MODEL_ID] for model in models if model.get(cst.MODEL_ID, None) is not None]
```

Or something similar right? What are the values of this models dict? We have dict[str, Any[ above which is dangerous